jscomp/main/cmij_main.ml

```diff
@@ -58,8 +58,8 @@ let from_cmj (files : string list) (output_file : string) : unit =
     Ext_list.map files (fun file -> 
         let module_name = (cmp file) in 
         let content : Js_cmj_format.t = Js_cmj_format.from_file file in 
-        assert (content.js_file_kind = Little_js);
-        assert (content.package_spec = Js_packages_info.runtime_package_specs);
+        (* assert (content.js_file_kind = Little_js); *)
+        (* assert (content.package_spec = Js_packages_info.runtime_package_specs); *)
```

Comment:
remove these assertions is dangerous -- why I did this is that we have some knowledge about this so that cmj format is a bit more compact.